src/box/lua/info.c

```diff
@@ -650,6 +651,21 @@ lbox_schema_version(struct lua_State *L)
 	return 1;
 }
 
+/* gethostname() Lua interface inside box.info. */
+static int
+lbox_info_hostname(struct lua_State *L)
+{
+	char buffer[_SC_HOST_NAME_MAX + 1];
```

Comment:
It is not a value of max hostname length, it is ID of the corresponding parameter in sysconf.

```sh
$ cc --std=c89 -Wall -Wextra -x c <(echo -e '#include <stdio.h>\n#include<unistd.h>\n\nint main() { printf("_SC_HOST_NAME_MAX: %d\\n", _SC_HOST_NAME_MAX); return 0; }') && ./a.out; rm a.out

_SC_HOST_NAME_MAX: 180
```

/usr/include/bits/confname.h:

```c
/* Values for the NAME argument to `pathconf' and `fpathconf'.  */
enum
  {
    <...>
    _SC_HOST_NAME_MAX,
#define _SC_HOST_NAME_MAX               _SC_HOST_NAME_MAX
```

Reply:
As expected, the mac has 255 default:
```
sysconf(_SC_HOST_NAME_MAX): 255
```

Reply:
Same for FreeBSD:
```
sysconf(_SC_HOST_NAME_MAX): 255
```

Reply:
Added `enum` constant.